src/layouts/Gallery.css

```diff
@@ -1,8 +1,14 @@
-.Gallery {
+/* This is to fix the size of the right section, .Gallery will take care of the flex and padding */
+.GalleryContainer {
     color: aliceblue;
-    width: 100vw;
+    width: calc(100% - 300px);
     min-height: 100vh;
     background-color: #3d4e85; /* Black */
+    margin-left: 300px;
+}
+
+.Gallery {
     flex-direction: row;
-    padding: 40px 350px;
+    display: flex; /* otherwise, the default display is flow */
```

Comment:
hmm, so this has the items auto-line-up nicely?

Reply:
flex-direction only takes effect if display is set to flex first

Reply:
ahh... that makes a lot of sense...